core/src/main/java/org/opensearch/sql/expression/operator/predicate/UnaryPredicateOperator.java

```diff
@@ -47,18 +43,12 @@ public static void register(BuiltinFunctionRepository repository) {
   }
 
   private static DefaultFunctionResolver not() {
-    return FunctionDSL.define(BuiltinFunctionName.NOT.getName(), FunctionDSL
-        .impl(UnaryPredicateOperator::not, BOOLEAN, BOOLEAN));
+    return FunctionDSL.define(
+        BuiltinFunctionName.NOT.getName(),
+        FunctionDSL.impl(UnaryPredicateOperator::not, BOOLEAN, BOOLEAN));
   }
 
-  /**
-   * The not logic.
-   * A       NOT A
-   * TRUE    FALSE
-   * FALSE   TRUE
-   * NULL    NULL
-   * MISSING MISSING
-   */
+  /** The not logic. A NOT A TRUE FALSE FALSE TRUE NULL NULL MISSING MISSING */
```

Comment:
LOL a not a true false false true null null missing missing.
Needs tags as well